### PR TITLE
Add Import and Export command

### LINKED ISSUE #119: Add Export Command

### PR DESCRIPTION
**Pull request by aexolate**
This PR adds the `import` and `export` command. The export command will write the data file into the specified location. The import command will read the data file from the specified location.

Also adds GUI option to import and export via the `File` > Import/Export button.

Closes #119

### CODE REVIEW

**Comment by aexolate** [comment]
CodeCov CI fails because of MainWindow changes

File: src/main/java/seedu/address/logic/commands/ImportCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+/**
+ * Imports a data file from the specified path.
+ */
+public class ImportCommand extends Command {
+
+    public static final String COMMAND_WORD = "import";
+    public static final String MESSAGE_SUCCESS = "Data successfully imported";
+    public static final String MESSAGE_FILE_ERROR = "An error occurred during loading of data. The file "
+            + "might be corrupted.";
+    public static final String MESSAGE_FILE_MISSING = "The specified file does not exists";
```
Comment (Goh-Li-Ting) [inline src/main/java/seedu/address/logic/commands/ImportCommand.java:22]: Should it be "exist" instead of "exists"?

**Review by Goh-Li-Ting — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/ImportCommand.java (hunk 1 above)
Comment (aexolate) [reply src/main/java/seedu/address/logic/commands/ImportCommand.java]: Fixed

**Review by Weiennn — APPROVED** [review]
LGTM

**Pull request merged**